api/reporting/schemas/input/descriptions/workspace.yaml

```diff
@@ -14,6 +14,9 @@ workspace:
       User ID of user who initially created this workspace.
   disseminate_research_other: |
       Description of user-defined research dissemination option.
+  free_tier_billing_usage: |
```

Comment:
This seems related to the old approach of joining it into the workspace table – remove?

Reply:
Thanks, reverted